src/pages/signing_official_console/DataCustodianTable.js

```diff
@@ -90,12 +92,12 @@ const IssueDataCustodianButton = (props) => {
 
 const researcherFilterFunction = getSearchFilterFunctions().signingOfficialResearchers;
 
-const CustodianCell = ({
+const SubmitterCell = ({
   researcher,
   showConfirmationModal
 }) => {
   const id = researcher.userId || researcher.email;
-  const button = researcher.isCustodian
+  const button = researcher.isDataSubmitter
```

Comment:
nit: would it be better to use a function, `isDataSubmitter(user)` here? I found the `setDataSubmitterProp` a bit unintuitive at first until I fully understood how it worked/interplayed with the rest of the system. i could also see someone in the future forgetting to call it, and not understanding why this button isn't showing up. totally understand if you'd rather just get it through, though.

Reply:
Yeah ... not proud of this bit of code. I was shooting for a quicker fix, but this is definitely a code smell. Will refactor that to be more consistent with how we're looking at other user roles.